ci-operator/step-registry/openshift/image-based/network-configuration/configure/openshift-image-based-network-configuration-configure-commands.sh

```
@@ -37,8 +61,9 @@ EOF
 
 chmod +x ${SHARED_DIR}/network-configuration.sh
 
-echo "Transfering network configuration script..."
-scp "${SSHOPTS[@]}" ${SHARED_DIR}/network-configuration.sh $ssh_host_ip:$remote_workdir
+echo "Transferring network configuration script..."
+ssh_retry scp "${SSHOPTS[@]}" ${SHARED_DIR}/network-configuration.sh $ssh_host_ip:$remote_workdir
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)